scripts/configure-flannel-nodes.sh

```
@@ -0,0 +1,149 @@
+    # Get all nodes
+    local nodes=$(oc get nodes -o jsonpath='{.items[*].metadata.name}' 2>/dev/null)
+    if [ -z "$nodes" ]; then
+        log [ERROR] "No nodes found in the cluster"
+        if [[ "${DPF_CLUSTER_TYPE}" == "hypershift" ]]; then
+            export KUBECONFIG="${saved_kubeconfig}"
+        fi
+        exit 1
+    fi
+    
+    # Convert to array
+    local node_array=($nodes)
```

Shellcheck (0.10.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)